src/Controls/tests/TestCases.HostApp/Issues/Issue21675.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using Microsoft.Maui.Controls;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 21675, "MenuFlyoutItem stops working after navigating away from and back to page", PlatformAffected.UWP | PlatformAffected.macOS)]
+	public class Issue21675 : NavigationPage
+	{
+		public Issue21675() : base(new MainPage())
+		{
+
+		}
+	}
+
+	public class MainPage : ContentPage
```

Comment:
Can you move this class to be an internal class of `Issue21675` so it's not part of the `Maui.Controls.Sample.Issues` NS

Reply:
@PureWeen, I have moved these classes to Issue21675.